guides/common/modules/con_puma-workers-and-threads-autotuning.adoc

```diff
@@ -23,3 +20,16 @@ Each Puma worker consumes around 1 GiB of RAM.
 ----
 # systemctl status foreman.service
 ----
+
+== Manually tuning Puma workers and threads count
```

Comment:
Please move this to a separate file. We have a convention to only have one (level1) heading per file. You can then include the file and use a leveloffset. Please also add an anchor and ensure title, anchor, and file name matches (file name has also a "type" prefix such as con_ or proc_).

Reply:
Moved, thank you!